docs/reference/method/MongoDBCollection-watch.txt

```diff
@@ -1,6 +1,8 @@
-================================
+============================
 MongoDB\\Collection::watch()
-================================
+============================
```

Comment:
I didn't notice build warnings for the previous header lines (not sure if strict warnings are being suppressed), but it is something RST can sometimes complain about so this was a quick fix.